packages/blockchain-ui/src/theme/components.ts

```diff
@@ -272,8 +270,7 @@ const buildComponentTheme = (colorPalette?: IColorPalette) => {
       },
       box: {
         common: {
-          border: `1px solid ${colors.grey300}`,
-          color: colors.grey500,
+          outlined: colors.grey500Opacity16,
```

Comment:
use alpha function as outlined above to change opacity of color

Reply:
Its Done..